zrml/swaps/src/mock.rs

```diff
@@ -1,27 +1,19 @@
 use crate as zrml_swaps;
-use frame_support::{construct_runtime, parameter_types};
+use frame_support::construct_runtime;
+use frame_support::parameter_types;
 use orml_currencies::BasicCurrencyAdapter;
 use orml_traits::parameter_type_with_key;
 use sp_runtime::{
     testing::Header,
-    traits::{AccountIdConversion, BlakeTwo256, IdentityLookup},
+    traits::AccountIdConversion,
+    traits::{BlakeTwo256, IdentityLookup},
     ModuleId,
 };
 use zeitgeist_primitives::{
     AccountIdTest, Amount, Asset, Balance, BlockNumber, BlockTest, CurrencyId, Hash, Index,
     MarketId, UncheckedExtrinsicTest, BASE,
 };
 
-pub const ALICE: AccountIdTest = 0;
-pub const BOB: AccountIdTest = 1;
-pub const CHARLIE: AccountIdTest = 2;
-pub const DAVE: AccountIdTest = 3;
-pub const EVE: AccountIdTest = 4;
-
-pub type AdaptedBasicCurrency = BasicCurrencyAdapter<Runtime, Balances, i128, u128>;
-pub type Block = BlockTest<Runtime>;
-pub type UncheckedExtrinsic = UncheckedExtrinsicTest<Runtime>;
-
```

Comment:
By looking into the diff, it seems that nothing has changed (no new type or excluded export). In case, I personally don't mind this rearrangement